lucid_work/visualizations.py

```diff
@@ -0,0 +1,158 @@
+# This file creates 3D visualizations of our DLA CNN model
+
+import numpy as np
+import tensorflow as tf
+import scipy.ndimage as nd
+import time
+import datetime
+import imageio
+import os
+import errno
+
+import lucid.modelzoo.vision_models as models
+from lucid.misc.io import show
+import lucid.optvis.objectives as objectives
+import lucid.optvis.param as param
+import lucid.optvis.render as render
+import lucid.optvis.transform as transform
+from lucid.modelzoo.vision_base import Model
+
+THRESHOLDS = (256,)
+TRANSFORMS = [transform.pad(4), transform.jitter(4)]
+
+
+LAYERS = { 'conv1': ['Conv2D', 100],
+           'conv1_relu': ['Relu', 100],
+           'pool1': ['MaxPool', 100],
+           'conv2': ['Conv2D_1', 96],
+           'conv2_relu': ['Relu_1', 96],
+           'pool2': ['MaxPool_1', 96],
+           'conv3': ['Conv2D_2', 96],
+           'conv3_relu': ['Relu_2', 96],
+           'pool3': ['MaxPool_2', 96]}
+
+
+
+class DLA(Model):
```

Comment:
This looks the same as above.  Maybe it should
be its own, small module to avoid code duplication?

Same with `LAYERS` and `TRANSFORMS`